### PR TITLE
Spec overview table formatting

### LINKED ISSUE #104: Make column spacing consistent when first spec added
The column spacing changes slightly between the state of having no specifications, and >=1 specification(s). It's visually a bit jarring. These were taken with the same browser viewport before and after.

no-specs

one-spec

### PR DESCRIPTION
**Pull request by catenoid**
Close #104, purely cosmetic.

Add explicit column widths and rotate chevron on click

### CODE REVIEW

File: frontend/src/app/specification-overview/specification-overview.component.css [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+#title-column {
+  width: 60%;
+}
+
+#version-column {
+  width: 20%;
+}
+
+#actions-column {
+  width: 20%;
```
Comment (fdw) [inline frontend/src/app/specification-overview/specification-overview.component.css]: Can we tell it to make this as small as possible, and use the rest for the other columns?

**Comment by fdw** [comment]
This sounds useful, too: https://stackoverflow.com/a/15884087

File: frontend/src/app/specification-overview/specification-overview.component.css (hunk 1 above)
Comment (catenoid) [reply frontend/src/app/specification-overview/specification-overview.component.css]: I'm a little uneasy about mixing up pixel and percent measurements. (which I assume was the proposition, as the oi buttons are of fixed size?) After searching I couldn't find a clear consensus as to whether it was safe. Also, the button to edit permissions on a specification conditionally appears based on whether the logged in user has those access rights, so I did not see this column as having an objective minimum width.

Comment (fdw) [reply frontend/src/app/specification-overview/specification-overview.component.css]: I'd probably have tried to set it to some ridiculous amount like `5%` :)

**Pull request merged**